cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/util/JSONUtils.java

```diff
@@ -34,7 +34,9 @@
 
     private static final String LOG_TAG = "JSONUtils";
 
-    public static final byte[] EMPTY_JSON = "{}".getBytes(Charset.forName("UTF-8"));
+    public static byte[] emptyJSONAsBytes(){
+        return "{}".getBytes(Charset.forName("UTF-8"));
+    }
```

Comment:
This isn't empty JSON, it's an empty JSON object (see suggested name in snippet below).

Can you be a bit more efficient here (and is FindBugs smart enough to accept), something like:

```java
private static final byte[] EMPTY_JSON = "{}".getBytes(Charset.forName("UTF-8"));

public static byte[] emptyJSONObjectAsBytes() {
    byte[] copy = new byte[EMPTY_JSON.length];
    System.arraycopy(EMPTY_JSON, 0, copy, 0, EMPTY_JSON.length);
    return copy;
}
```